src/sentry/db/models/fields/slug.py

```diff
@@ -6,6 +6,8 @@ def no_numeric_validator(value: str) -> None:
     if value.isnumeric():
         raise ValidationError("Value cannot be entirely numeric.")
 
+    return None
```

Comment:
this isn't what the linter was suggesting -- you shouldn't need this

it was saying you were *assigning* the return value:

```python
def f() -> None:
    ....

x = f()  # the type checker is telling you this is probably a mistake because `f` cannot return anything
```

Reply:
Ahh thank you, makes sense! I got rid of the function so no longer an issue, but good to know that's what the linter was trying to communicate